stack/lab/Dockerfile

```diff
@@ -41,19 +41,17 @@ ARG AIIDALAB_HOME_VERSION
 RUN git clone https://github.com/aiidalab/aiidalab-home && \
      cd aiidalab-home && \
      git checkout v"${AIIDALAB_HOME_VERSION}" && \
-     pip install --quiet --no-cache-dir "./" && \
+     pip install --no-user --quiet --no-cache-dir "./" && \
```

Comment:
With `--no-user` the package will be installed to where?

Reply:
`--no-user` simply means default pip behaviour, so in this case it will be installed to `/opt/conda`, as before.

Reply:
https://pip.pypa.io/en/stable/topics/configuration/#environment-variables